.github/workflows/no-response.yml

```diff
@@ -0,0 +1,23 @@
+name: No Response Bot
+
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # "every 6 hours"
+    - cron: '0 */6 * * *'
```

Comment:
To decrease CI overload and match our current behavior, please change this to "every 24h".

Reply:
oh sure! Updated in https://github.com/microsoft/LightGBM/pull/5120/commits/d256695133ec16cb348fc203240910bfe46f1eb7.

I picked an arbitrary hour (04:00 UTC). If you have any strong opinion about running at a particular time of day, I'd be happy to change it.